packages/studio/src/store/slices/accountContent/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { EntitiesResponse, MessageID, ResponseType } from "@yext/studio-plugin";
+import sendMessage from "../../../messaging/sendMessage";
+
+export async function fetchSavedFilters() {
+  const savedFiltersResponse = await sendMessage(
+    MessageID.GetSavedFilters,
+    null,
+    { hideSuccessToast: true }
+  );
+  return savedFiltersResponse.type === ResponseType.Success
+    ? savedFiltersResponse.savedFilters
+    : [];
+}
+
+export async function fetchEntitiesRecord() {
+  const entityTypesResponse = await sendMessage(
+    MessageID.GetEntityTypes,
+    null,
+    { hideSuccessToast: true }
+  );
+  if (entityTypesResponse.type !== ResponseType.Success) {
+    return {};
```

Comment:
Should we log an error?

Reply:
An error will already be logged in the terminal if there was an error getting the response (e.g. https://github.com/yext/studio/blob/feature/add-page-improvements/packages/studio-plugin/src/messaging/registerGetSavedFilters.ts#L22)

Reply:
I still think it'd be nice to log an error in the browser. I'm not sure if studio in storm will show the terminal output. When I implemented a similar fetchEntitiesRecord for my item the console.error was also nice when running jest tests to tell me something was wrong

Reply:
Added!